scripts/deploy-gateway.js

```diff
@@ -10,27 +10,45 @@ const {
 } = require('ethers');
 
 const { execSync }  = require('child_process');
+const path = require('node:path');
+
+const {
+  printLog,
+  printObj,
+} = require('./logging');
 
 // these environment variables should be defined in an '.env' file
+const contractsPath = process.env.CONTRACTS_PATH || '../build';
 const prefix = process.env.PREFIX;
 const chain = process.env.CHAIN;
 const url = process.env.URL;
 const privKey = process.env.PRIVATE_KEY;
 const adminThreshold = parseInt(process.env.ADMIN_THRESHOLD);
 
-const provider = new JsonRpcProvider(url);
-const wallet = new Wallet(privKey, provider);
+printObj({"environment_variables:" : {
+  "CONTRACTS_PATH" : contractsPath || null,
```

Comment:
If they are `undefined`, the fields will omitted from the output. I wanted to force those fields to show up regardless, so that users know which variables are missing